pkgs/desktops/lxde/core/lxsession/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchgit
+, libtool
+, automake
+, autoconf
+, intltool
+, pkg-config
+, libxml2
+, libxslt
+, docbook_xml_dtd_412 
+, docbook_xsl
+, wrapGAppsHook
+, glib
+, libX11
+, gtk2-x11
+, polkit
+, vala
+}:
+let
+  version = "0.5.5";
+in
+stdenv.mkDerivation {
+
+  pname = "lxsession";
+  inherit version;
+
+  src = fetchgit {
+    url = "git://git.lxde.org/git/lxde/lxsession";
+    rev = version;
+    sha256 = "17sqsx57ymrimm5jfmcyrp7b0nzi41bcvpxsqckmwbhl19g6c17d";
+  };
+
+  patches = [ ./xmlcatalog_patch.patch ];
+
+  nativeBuildInputs = [
```

Comment:
Sort inputs alphabetically

Reply:
Sorted alpbetically nativeBuildInputs together and buildInputs together.